lms/djangoapps/grades/models.py

```diff
@@ -403,7 +386,6 @@ def create_grade(cls, **params):
         cls._prepare_params_and_visible_blocks(params)
         cls._prepare_attempted_for_create(params, now())
         grade = cls(**params)
-        grade.full_clean()
         grade.save()
```

Comment:
Nit: Since we no longer need the intermediary step, also revert the code change and reduce the above 2 lines to:
`grade = cls.objects.create(**params)`